samples/python2/_tutorials_url.txt

```diff
@@ -0,0 +1,2 @@
+OpenCV-Python Tutorials
+http://opencv-python-tutroals.readthedocs.org/en/latest/py_tutorials/py_tutorials.html
```

Comment:
AFAIK, these tutorials has been transferred to OpenCV and are no longer maintained. So this link can be removed.